IdentityCore/src/requests/MSIDSilentTokenRequest.h

```diff
@@ -34,12 +34,19 @@
 @class MSIDExternalAADCacheSeeder;
 #endif
 
+@protocol MSIDSilentTokenRequestDelegate <NSObject>
+
+-(BOOL)skipCahcedRefreshToken;
```

Comment:
nit: typo and missing space?
```suggestion
- (BOOL)skipCachedRefreshToken;
```